src/Session/EncryptedStore.php

```
@@ -32,10 +32,14 @@ protected function prepareForStorage($data)
 
     protected function prepareForUnserialize($data)
     {
+        if ($data === '') {
+            return '';
+        }
+
         try {
             return $this->encrypter->decrypt($data);
         } catch (Exception $e) {
-            $this->exceptionHandler->report($e);
+            $this->exceptionHandler?->report($e);
```

Should we check for an exception handler and otherwise allow the error to propagate? If this fails silently it could be a nasty bug to find.